advanced/lighttpd.conf.debian

```diff
@@ -69,7 +69,7 @@ $HTTP["url"] =~ "^/admin/" {
         "X-Frame-Options" => "DENY"
     )
 
-    $HTTP["url"] =~ ".ttf$" {
+    $HTTP["url"] =~ "\.(eot|otf|tt[cf]|woff2?)$" {
```

Comment:
Little question: Since this PR is to tweak the blocking page and also touches only files of the blocking page, why does it include adding access for further fonts file types for the admin panel? As it's inside the `$HTTP["url"] =~ "^/admin/"` block, it should not have an effect on the blocking page?

I'm asking, since we would need to make this change for our DietPi webserver configs as well and I stumbled over this since there was no such directive in our `$HTTP["url"] =~ "^/pihole/"` block, nor is it upstream 😉.

Reply:
Because in AdminLTE develop branch we switched to modern fonts https://github.com/pi-hole/AdminLTE/tree/devel/style/vendor/SourceSansPro

Reply:
I simply changed this to include more font types. I don't have the time to look if this is even needed and that should be made in another PR.

Reply:
Okay, so it's not directly related to this PR but thanks for the link to AdminLTE, that is reason enough to add it to our configs as well 👍.

Reply:
Actually the comment below "`# Allow Block Page access to local fonts`" is wrong then, as its the admin panel, isn't it?

Reply:
Like I said, this PR does not change the code apart from the fonts. If you have further questions you should open a new issue.